core/http/endpoints/localai/metrics.go

```diff
@@ -0,0 +1,42 @@
+package localai
+
+import (
+	"time"
+
+	"github.com/go-skynet/LocalAI/core/services"
+	"github.com/gofiber/fiber/v2"
+	"github.com/gofiber/fiber/v2/middleware/adaptor"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+func LocalAiMetricsEndpoint() fiber.Handler {
+	return adaptor.HTTPHandler(promhttp.Handler())
+}
+
+type apiMiddlewareConfig struct {
+	Filter         func(c *fiber.Ctx) bool
+	metricsService *services.LocalAIMetricsService
+}
+
+func LocalAiMetricsAPIMiddleware(metrics *services.LocalAIMetricsService) fiber.Handler {

```

Comment:
```suggestion
func LocalAIMetricsAPIMiddleware(metrics *services.LocalAIMetricsService) fiber.Handler {
```
for consistency, in Golang it is used to have this notation to not break acronyms. For instance,

ReadYAML, ReadJSON, etc. 

likewise would apply to LocalAI : LocalAI instead of LocalAi, and so on so forth